ios/Podfile.lock

```diff
@@ -530,4 +530,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: d641f53b267b5fdc2398388c8211cf0739401d29
 
-COCOAPODS: 1.9.2
```

Comment:
Should we document the why of this downgrade?

Reply:
Can you actually help me understand why this happened?

Reply:
Oh it's probably an outdated version cocoapods 🤔 maybe running `gem install cocoapods` will take care of it?